.travis.yml

```diff
@@ -32,7 +32,10 @@ before_script:
   - vendor/bin/phinx --configuration=phinx.yml.dist migrate -e testing
 
 script:
-  - vendor/bin/phpunit --coverage-clover build/logs/clover.xml
+  - ./vendor/bin/phpunit --coverage-clover build/logs/clover.xml
+  - ./vendor/bin/php-cs-fixer fix classes/  --verbose --level=psr2 --dry-run
+  - ./vendor/bin/php-cs-fixer fix tests/  --verbose --level=psr2 --dry-run
```

Comment:
@yitznewton 

Instead of having two runs, you can also specify a configuration file in which you can

* configure a `Finder` to use for iterating over (directory to start in, exclusions, extensions, etc.)
* base fixer level
* additional fixers
* excluded fixers